localstack/services/apigateway/helpers.py

```diff
@@ -0,0 +1,30 @@
+import re
```

Comment:
I though it might be good extracting the the functions that somehow looks as helper functions, so I started with the two functions that I needed to reuse them in tests. I think this can improve/increase the maintainability a bit somehow.